RELEASE.md

```diff
@@ -50,3 +50,23 @@ WIP: this document is still incomplete.
     TODO: we should have an automation KFP cluster, and the waiting and submiting second cloudbuild task should be automated.
 1. Release `kfp-server-api` and `kfp` python packages on pypi.
 1. Create a github release using `$VERSION` git tag, fill in the description.
+
+## Cherry picking PRs to release branch
+
+### Option - Kubeflow cherry_pick_pull.sh helper
```

Comment:
Is the title only caveat of this cherry pick script? If we don't know the full caveats of this script, we probably don't need to list if here. Otherwise, we probably should have a more certain or precise statement like using the script and revising the title if needed, so users can have more confidence of trying out this script.

Reply:
Good point! I'll move the option below the other one and not recommend it by default.